### PR TITLE
Add sanity-seo-pane plugin for SEO settings

### PR DESCRIPTION
**Pull request by mamfi-0221**
initial commit adding sanity-seo-pane plugin

### CODE REVIEW

File: schemas/documents/seo.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import { MdSearch } from "react-icons/md";
```
Comment (dorelljames) [inline schemas/documents/seo.js:1]: Please move this to `objects` directory instead of `documents`.

File: src/resolvePreviewUrl.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 export default function resolveProductionUrl(document) {
-  const previewUrl = `api/preview?secret=secret&slug=${document.slug.current}`
+  const previewUrl = `api/preview?secret=secret&slug=${document.slug.current}&fetch=true`
```
Comment (dorelljames) [inline src/resolvePreviewUrl.js:2]: I realized `&fetch=true` isn't needed at all, so please remove.

File: schemas/documents/seo.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  fields: [
+    {
+      title: "SEO Title",
+      name: "seoTitle",
+      description: "This will override the page title",
+      type: "string"
+    },
+    {
+      title: "Keywords",
+      name: "keywords",
```
Comment (dorelljames) [inline schemas/documents/seo.js:17]: can we make it all uniform by prefixing names with `seo`, so in this case, it should be `seoKeywords`. Use `camelCase` as well.

File: schemas/documents/seo.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+      description: "Separate with commas",
+      type: "string"
+    },
+    {
+      title: "Synonyms",
+      name: "synonyms",
```
Comment (dorelljames) [inline schemas/documents/seo.js:23]: `seoSynonyms`

File: schemas/documents/seo.js [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+      description: "These are similar words to inform the SEO review",
+      type: "string"
+    },
+    {
+      title: "Description",
+      name: "description",
```
Comment (dorelljames) [inline schemas/documents/seo.js:29]: `seoDescription`

**Review by dorelljames — CHANGES_REQUESTED** [review]

File: schemas/documents/seo.js (hunk 1 above)
Comment (mamfi-0221) [reply schemas/documents/seo.js]: Moved `seo` schema to objects folder.

File: src/resolvePreviewUrl.js (hunk 2 above)
Comment (mamfi-0221) [reply src/resolvePreviewUrl.js]: removed url parameter

File: schemas/documents/seo.js (hunk 3 above)
Comment (mamfi-0221) [reply schemas/documents/seo.js]: changed `keywords` to `seoKeywords`

File: schemas/documents/seo.js (hunk 4 above)
Comment (mamfi-0221) [reply schemas/documents/seo.js]: changed `synonyms` to seoSynonyms

File: schemas/documents/seo.js (hunk 5 above)
Comment (mamfi-0221) [reply schemas/documents/seo.js]: changed `description `to `seoDescription`

Comment (dorelljames) [reply schemas/documents/seo.js]: thanks

File: schemas/documents/seo.js (hunk 4 above)
Comment (dorelljames) [reply schemas/documents/seo.js]: awesome!

File: schemas/documents/seo.js (hunk 1 above)
Comment (dorelljames) [reply schemas/documents/seo.js]: awesome. thanks

**Review by dorelljames — APPROVED** [review]
Thanks! Ready to merge.

**Pull request merged**